ipstools.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
```

Comment:
missing newline

Reply:
Where? This example is a unmodified copy of the "Setuptools Getting Started Example"...

Reply:
Normally, the last character of a text file should be a newline. Github shows this missing newline symbol and in magit you can see it too. It's just a minor nitpick.